content/pt/docs/concepts/resources/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Resources
+weight: 70
+default_lang_commit: a1740fd934e595f1396f2eb82a58a80824369b09
+---
+
+## Introdução
+
+{{% docs/languages/resources-intro %}}
+
+Se você usar [Jaeger](https://www.jaegertracing.io/) como seu backend de
+observabilidade, os _resource attributes_ serão agrupados na guia **Process**:
+
+![Uma captura de tela do Jaeger mostrando um exemplo de saída de resource attributes associados a um rastro.](screenshot-jaeger-resources.png)
+
+Um _resource_ é adicionado ao `TraceProvider` ou `MetricProvider` quando eles
+são criados durante a inicialização. Esta associação não pode ser alterada
+posteriormente. Após um _resource_ ser adicionado, todos os trechos e métricas
```

Comment:
```suggestion
posteriormente. Após um resource ser adicionado, todos os trechos e métricas
```

Só precisa colocar em itálico quando a palavra aparece pela primeira vez, não há a necessidade de deixar a palavra em itálico se elas se repete no texto.

Reply:
O mesmo vale para outras palavras não traduzidas e que se repetem no texto.

Reply:
I wouldn't use italics at all.